pkg/kubectl/cmd/cmd.go

```diff
@@ -110,23 +110,26 @@ __custom_func() {
 }
 `
 	valid_resources = `Valid resource types include:
-   * pods (aka 'po')
-   * replicationcontrollers (aka 'rc')
+   * componentstatuses (aka 'cs')
    * daemonsets (aka 'ds')
-   * services (aka 'svc')
+   * deployments
```

Comment:
Yeah, was about to write the same thing. I'll go with the released resources then.